stable/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 2.1.2
+version: 2.1.3
```

Comment:
This adds a new feature, so should bump the minor version.

Reply:
✅